assets/pages/oneoff-contributions/helpers/validation.js

```diff
@@ -0,0 +1,30 @@
+// @flow
```

Comment:
Like PaymentMethods, this file is really similar to (this one)[https://github.com/guardian/support-frontend/blob/master/assets/pages/monthly-contributions/helpers/validation.js] We should create a shared helper so this can be reused.